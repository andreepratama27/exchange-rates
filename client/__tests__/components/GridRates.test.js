import React from "react";
import { shallow } from "enzyme";
import "./../jestsetup";

describe("Testing grid", () => {
  test("render text", () => {
    expect(
      shallow(<p>render text</p>)
        .find("p")
        .exists()
    ).toBe(true);
  });

  test("render bold text", () => {
    expect(<b>Bold text</b>).toMatchSnapshot();
  });
});
