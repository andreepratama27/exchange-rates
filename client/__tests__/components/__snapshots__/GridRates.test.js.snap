// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing grid render bold text 1`] = `
<b>
  Bold text
</b>
`;
